drivers/hmis/app/graphql/schema.graphql

```diff
@@ -8,7 +8,7 @@ type Client {
   Date of birth as format yyyy-mm-dd
   """
   dob: ISO8601Date
-  enrollments: [Enrollment!]!
+  enrollments(limit: Int, offset: Int): EnrollmentsPaginated!
```

Comment:
I think it's correct that limit and offset are optional, but some default values should be set for them.


Offset looks like it should be set to default `0` in the `PaginatedScope` initializer but for some reason that's not working, I still get the error `Cannot return null for non-nullable field EnrollmentsPaginated.offset`. The default for `limit` can be 50.

Example of query that should work, but is erroring:
```
{
  client(id: 256) {
    id
    enrollments {
      limit
      offset
      nodes {
        id
      }
    }
  }
}

```

Reply:
I'll double check that it's wired up correctly under the hood

Reply:
@gigxz This should be fixed now